### PR TITLE
Added BIOS versions and documentation for various motherboards located in pc, at and genpc.cpp

### PR DESCRIPTION
**Pull request by rfka01**
pc:	New machines marked as NOT_WORKING
Compaq Deskpro [jafir, rfka01]

genpc:	
Added generic AMI XT ROM BIOS [PCem, rfka01]
Added VIP M X M/10 BIOS	[Predator99, rfka01]
Added V20-BIOS Version 3.65 (C) Peter Köhlmann 1987	[Predator99, rfka01]

at:	New machines marked as NOT_WORKING
PC-Chips M209 motherboard (286)	[Am386DX-40, rfka01]
Siemens PCD-2M [Predator99, rfka01]
Octek Fox M 286	[HanJammer, rfka01]
Wearnes CL286-12/16S (CL286-12S and CL286-16S) [HanJammer, rfka01]
ASI 100B0, aka "HAM 12 TI 286 Motherboard ZERO WAIT" [CMB75, rfka01]
Advanced Logic Research PWB 7270 REV E [lgj1107, rfka01]
CAF Prolite 286/16 [jensma, rfka01]
Twinhead Netstation PC [HanJammer, rfka01]
ICL DRS M40 (286) [Deksor, rfka01]
Peacock 286-S [Predator99, rfka01]
SIIG MiniSys 2000 [zoinknoise, rfka01]
ECS 8517 v3.3 (386sx) [Horun, rfka01]
Diamond Flower International 386SX-16/20CN Rev 1.0 [Vipersan, rfka01]
Morse KP920121523 V2.20 (386) [Predator99, rfka01]
SOYO Baby AT 386 [Aerion, rfka01]
Philips P3239 [Predator99, rfka01]
Packard Bell PCB-303 Rev.01	[Predator99, rfka01]
FIC 386-SC-HG [Predator99, rfka01]
CACHING TECH CORPORATION C386MX	[p1p1p1, rfka01]
Toshiba T3200SXC [adalbert, rfka01]
Peacock P386DX-40 [Predator99, rfka01]
Commodore 386SX-25 [sdose, rfka01]
PC-Chips M317 motherboard (386) [Predator99, rfka01]
ZEOS 386 SX-16 [Horun, rfka01]
Quadtel QTC-SXM KT X20T02/HI Rev.3 [Thermalwrong, rfka01]
ELT-P9 [Predator99, rfka01]
ECS (Elitegroup) 386A [TheMobRules, rfka01]
Morse M3 V3.00 [Am386DX-40, rfka01]
Prolink P386SX-25PW VER:2.00 [Eep386, rfka01]
ILON USA, INC. M-396B (386sx) [BreakPoint, rfka01]
486MMBO4088 (486SLC) [alvaro84, rfka01]
80386SX-VH-COM [Predator99, rfka01]
DTK MBA-032Q TK83305-4N-D-03 [SSTV2, rfka01]
Amstrad PC9486 [Zalog, rfka01]
ASUS ISA-486 - Rev. 1.4	[TheMobRules, rfka01]
ASUS ISA-486SIO rev. 1.2 [BreakPoint, rfka01]
Micronics JX30GP [SSTV2, rfka01]
FIC 486-GVT	[zoinknoise, rfka01]
MSI MS-4132 G VER:1	[TheMobRules, rfka01]
ADI F4DXL-UC4 [Predator99, rfka01]
Silicon Valley Computer, Inc. 486WB6A3.B1 [zero01101, rfka01
NEC APC IV [jafir, dhau, rfka01]
ABIT AB-AX4	[fool, rfka01]
Jamicon KM-S4-1 VER 1.1	[BreakPoint, rfka01]
Gemlight GMB-486SG rev 2.2 [Deksor, rfka01]
Soyo 025K2 [Ph@ntom-X, rfka01]
Gemlight GMB-486UNP v2.1 [Deksor, rfka01]
BIOSTAR - MB-1433/50 AEA-P - V:1 [Thermalwrong, rfka01]
FIC 486-VC-HD [Darkscop, rfka01]
FIC ELI6-II (from Unisys ELI 46665 Desktop)	[Horun, rfka01]
Mitac PWA-IH4077C [Horun, rfka01]
Mitac PWA-IH4077D [8bitbubsy, rfka01]
Mitac MBA-029 [Madao, rfka01]
Chaintech 4SPI [AskNotes, rfka01]
Addtech Research 4GLX3 Green-B 4GPV3.1 aka VisionEX 4GPV3 [Horun, rfka01]
TMC PAT48AV 1.4	[Madao, rfka01]
Jetway J-403TG [BreakPoint, computerguy08, theoryboy, rfka01]
TMC PAT48PG4-V1.20 [HanJammer, rfka01]
Abit AB-AV4 (aka VL-BUS 486) [HanJammer, rfka01]
Lucky Star UCM-486V30 (aka SIS486 3-VLBUS) [BreakPoint, rfka01]
A-Trend ATC-1425B [Windows9566, rfka01]
SOYO 486 VESA 025D2	[BreakPoint, rfka01]
PowerTech MB457 aka Pine PT-2068.1 [BreakPoint, rfka01]

at:
Added Digicom DIGIS 286S Turbo8/10MHz BIOS [Predator99, rfka01]
Added ELITEGROUP COMPUTER SYSTEMS, INC. NEATSX Rev 1.0 BIOS	[jensma, rfka01]
Added H286-C3 158 BIOS [weedeewee, rfka01]
Added BIOS versions to the suntac6 driver [Predator99, rfka01]
Added BIOS version to the ht12a driver [zoinknoise, rfka01]
Added AT046DX3-B2.1(PQFP) BIOS version [lgj1107, rfka01]
Added BIOS version for C3404 Rev:B to the alim1419 driver [Horun, rfka01]
Added BIOS version for the FIC 386SC REV A2 MBZ86418 [Horun, rfka01]
Added PLATO TECHNOLOGY CO. 386 motherboard BIOS [Predator99, rfka01]
Added BIOS for Peacock 4386-VCHD [Predator99, rfka01]
Added PCChips M321 Rev.2.7 BIOS version	[alvaro84, rfka01]
Added BIOS version MR BIOS V1.44 to the um82c481af driver [alvaro84, rfka01]
Added BIOS version to the fic486kvd driver [Predator99, rfka01]
Added BIOS version  4.26GN2A to the ficvipio driver	[AskNotes, rfka01]
Added BIOS version to the abpi4 driver [BreakPoint, rfka01]
Added BIOS version to the zito4dps driver [BreakPoint, rfka01]
Added BIOS version to the zi4dvs driver	[Predator99, rfka01]
Added BIOS version  MB-4DUV/UVC VER 3.1 to the md4duvc driver [waterbeesje, rfka01]
Added BIOS version vBS1.10H UM8810 ECS (Highscreen boot logo) to the um8810paio driver [SSTV2, rfka01]
Added BIOS version UNIC2 94V-0 to the at486 driver [Distortions, rfka01]
Added BIOS version Tomato 4DPS v1.6 to the zito4dps driver [omarsis, rfka01]
Added Amptron DX9200 BIOS version to the pccm915i driver [Horun, rfka01]
Added BIOS version to the op82c392 driver [lgj1107, rfka01]
Added BIOS version to the um8498f driver [Deunan, rfka01]
Added BIOS versions to the at486 driver [lgj1107, rfka01]
Added BIOS version to the abax4 driver [Predator99, rfka01]
[Commits June 2021.xlsx](https://github.com/mamedev/mame/files/6795564/Commits.June.2021.xlsx)

### CODE REVIEW

**Comment by cuavas** [comment]
Removing blank lines in the comment at the top of at.cpp makes it harder to read.

File: src/mame/drivers/at.cpp [hunk 1]
```diff
@@ -3,7 +3,6 @@
 /***************************************************************************
 
     IBM AT Compatibles
-
```
Comment (cuavas) [inline src/mame/drivers/at.cpp:6]: This blank line was useful.

File: src/mame/drivers/at.cpp [hunk 2]
```diff
@@ -17,7 +16,6 @@ Video: PC 30-III: ATI EGA Wonder 800+, PC 40-III: Onboard Paradise VGA, 256KB
 Mass storage: One HD disk drive standard, second drive optional; PC 30-III: 20MB, PC 40-III: 40MB AT-IDE HD standard, 80MB or 100MB optional
 On board: Serial, Parallel, Commodore 1532 Mouse port (MS Bus mouse compatible), Keyboard, Beeper, Floppy (2 devices), AT-IDE (1 device)
 Options: 80287
-
```
Comment (cuavas) [inline src/mame/drivers/at.cpp:20]: This is another blank line before a heading that you’ve removed.

File: src/mame/drivers/at.cpp [hunk 3]
```diff
@@ -27,7 +25,6 @@ RAM: 1MB - 8MB on board
 Mass storage: 1.44MB Floppy disk drive and 80MB IDE hard disk
 On board: 2xserial, parallel, bus mouse, keyboard
 To-Do: Complains about missing mouse hardware (Bus Mouse), hangs in POST
-
```
Comment (cuavas) [inline src/mame/drivers/at.cpp:30]: This is yet another blank line before a heading that aided readability that you’ve removed.

File: src/mame/drivers/at.cpp [hunk 4]
```diff
@@ -38,7 +35,6 @@ Mass storage: 1.2MB Floppy disk drive and 20MB or 40MB MFM harddisk
 Bus: Vertical passive ISA backplane with six slots
 On board: 2xserial, parallel, floppy, keyboard, RTC, MFM harddisk controller piggybacked to bus extension on slot CPU
 Options: 80287
-
```
Comment (cuavas) [inline src/mame/drivers/at.cpp:41]: Yet another one.

File: src/mame/drivers/at.cpp [hunk 5]
```diff
@@ -51,7 +47,6 @@ Bus: two 8bit and two 16bit ISA slots
 On board: Serial, parallel
 Standard cards: Floppy/IDE combo card, special Compaq CGA/MDA hybrid video card
 Options: Compaq EGA card (drives internal monitor), 80287, floppy drives (360K, 1.2M, 1.44M)
-
```
Comment (cuavas) [inline src/mame/drivers/at.cpp:54]: And another.

File: src/mame/drivers/at.cpp [hunk 6]
```diff
@@ -64,7 +59,6 @@ Mass storage: One 1.2MB floppy disk drive, no/20MB/40MB hard disk
 On board: Serial, Parallel, RTC, RGBI (external Monitor), keyboard
 Options: 80827, Expansion box with 2 ISA slots, 300/1200Baud internal Modem, Compaq EGA Board
 To-Do: Emulate Graphics card fully
-
```
Comment (cuavas) [inline src/mame/drivers/at.cpp:67]: Yet another blank line before a heading.

File: src/mame/drivers/at.cpp [hunk 7]
```diff
@@ -73,7 +67,6 @@ Form factor: Desktop PC
 CPU: Intel 286, 8MHz
 RAM: 640KB
 Mass storage: Floppy: 5.25" 1.2Mb, HDD: 40Mb
-
```
Comment (cuavas) [inline src/mame/drivers/at.cpp:76]: Another one.

File: src/mame/drivers/at.cpp [hunk 8]
```diff
@@ -84,7 +77,6 @@ RAM: 512K on CPU card, 128K on a piggyback card and a memory expansion board
 Bus: Passive backplane, ISA
 Video: Paradise EGA on another piggyback board
 Mass storage: Floppy: 5.25" 1.2MB, MFM HDD
-
```
Comment (cuavas) [inline src/mame/drivers/at.cpp:87]: And still another.

File: src/mame/drivers/at.cpp [hunk 9]
```diff
@@ -3848,15 +4226,13 @@ ROM_START( at486 )
...
 	/* 486 boards from FIC
-
 	naming convention
 	xxxxx101 --> for EPROM
 	xxxxx701 --> for EEPROM using a Flash Utility v5.02
 	xxxxBxxx --> NS 311/312 IO Core Logic
 	xxxxCxxx --> NS 332 IO Core Logic
 	xxxxGxxx --> Winbond W83787F IO Core Logic
 	xxxxJxxx --> Winbond W83877F IO Core Logic
-
```
Comment (cuavas) [inline src/mame/drivers/at.cpp:3859]: There’s no reason to remove these blank lines.

File: src/mame/drivers/at.cpp [hunk 10]
```diff
@@ -5293,227 +5673,245 @@ ROM_END
 
 
 /***************************************************************************
-
   Game driver(s)
-
```
Comment (cuavas) [inline src/mame/drivers/at.cpp:5298]: Most heading comments like this have the blank lines between the rows of asterisks and the text.

**Review by cuavas — COMMENTED** [review]
I still get the feeling you’re not actually checking your diffs locally before pushing.

**Comment by rfka01** [comment]
I undid some work by copypasting from  the github page, this is probably where I lost the blank lines.

**Comment by rfka01** [comment]
That's all new BIOS versions I have for now - it would be great if this could be merged.

**Comment by cuavas** [comment]
I think @Tafoid was looking at this.  The trouble is, it’s now very big, and summarising the additional machines and BIOSes for the final squashed commit message was taking him some time.

**Pull request merged**